### PR TITLE
Targets Module

### PR DESCRIPTION
**Pull request by yulia-bel**
This PR adds Targets Module and tests.

### CODE REVIEW

File: api/src/modules/targets/targets.controller.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import {
```
Comment (tiagojsag) [inline api/src/modules/targets/targets.controller.ts:1]: image

Comment (tiagojsag) [reply api/src/modules/targets/targets.controller.ts:1]: test if it works like it does now without them. In some scenarios, things change.

File: api/src/modules/targets/targets.service.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public readonly targetsRepository: TargetsRepository,
+  ) {
+    super(
+      targetsRepository,
+      targetResource.name.singular,
+      targetResource.name.singular,
+    );
+  }
+
+  // To Do
```
Comment (tiagojsag) [inline api/src/modules/targets/targets.service.ts:31]: todo? what' missing?

File: api/test/integration/import-data/xlsx-uploads/.~lock.base-dataset.xlsx# [hunk 3]
```diff
@@ -0,0 +1 @@
+,yulia,omen,20.01.2022 17:46,file:///home/yulia/snap/libreoffice/239/.config/libreoffice/4;
```
Comment (tiagojsag) [inline api/test/integration/import-data/xlsx-uploads/.~lock.base-dataset.xlsx#:1]: image

**Review by tiagojsag — CHANGES_REQUESTED** [review]

File: api/test/integration/import-data/xlsx-uploads/.~lock.base-dataset.xlsx# (hunk 3 above)
Comment (yulia-bel) [reply api/test/integration/import-data/xlsx-uploads/.~lock.base-dataset.xlsx#]: cleaned the mess, sorry 😬

**Review by alexeh — COMMENTED** [review]
Sorry for being late to this CR.

Looks like most of the stuff has been properly reviewed / addressed

I just left a minor comment, let me know what you think about

Thanks!

**Review by alexeh — APPROVED** [review]
LGTM, thank you @yulia-bel!

**Pull request merged**